src/main/webapp/user_script.js

```diff
@@ -34,34 +34,61 @@ function loadProfile() {
   });
 }
 
+/**Fetches the items that were liked by user and sends the array to 
+* populate the page with the item cards.
+*/
 function loadFavItems() {
   fetch('/favorite-item')
     .then((response) => response.json())
     .then((favorites) => {
       var itemIdArray = new Array();
-      const itemId = favorites;
-      itemIdArray.push(itemId);
-      itemIdArray.forEach(getItem);
+      itemIdArray = favorites;
+      populateFavoriteItemGrid(itemIdArray)
```

Comment:
This can be simplified to:
`populateFavoriteItemGrid(favorites)`

The `itemIdArray` variable can be removed.